test/prob/test_fixture_cdf_log.hpp

```diff
@@ -354,11 +354,15 @@ class AgradCdfLogTestFixture : public ::testing::Test {
         << "Number of finite diff gradients and calculated gradients must "
            "match -- error in test fixture";
     for (size_t i = 0; i < finite_dif.size(); i++) {
-      EXPECT_NEAR(finite_dif[i], gradients[i], 1e-4)
-          << "Comparison of finite diff to calculated gradient failed for i="
-          << i << ": " << parameters << std::endl
-          << "  finite diffs: " << finite_dif << std::endl
-          << "  grads:        " << gradients;
+      // use relative error check in first case as logs of functions do not
+      // necessarily converge to asymptotic values
+      if (abs((finite_dif[i] - gradients[i]) / finite_dif[i]) > 0.01) {
```

Comment:
Just delete this extra code and the test that triggers it (the y = -50 example). I don't wanna worry about changing this test framework.

Weird to say delete a test, but lemme just justify that by saying:

I had a closer look today and I agree this totally looks like a shortcoming in the distribution test framework, not the code here.

Also, you have the test in test/unit/math/mix/scal/prob/normal_cdf_log_test.cpp that covers this exact case with Bob's autodiff stuff.

Reply:
Okay I've reverted back to the original file. I guess the relative error checks in the rev test should at least keep a record of the issue in the negative tail. Let me know if you need anything else!